store/postgres/src/relational_queries.rs

```diff
@@ -1230,50 +1230,52 @@ impl<'a> LoadQuery<PgConnection, EntityData> for FindManyQuery<'a> {
 
 impl<'a, Conn> RunQueryDsl<Conn> for FindManyQuery<'a> {}
 
-#[derive(Debug, Clone)]
+#[derive(Debug)]
 pub struct InsertQuery<'a> {
     table: &'a Table,
-    key: &'a EntityKey,
-    entity: Entity,
+    entities: &'a mut Vec<(EntityKey, Entity)>,
     block: BlockNumber,
 }
 
 impl<'a> InsertQuery<'a> {
     pub fn new(
         table: &'a Table,
-        key: &'a EntityKey,
-        entity: Entity,
+        _entity_type: &'a EntityType,
```

Comment:
If we are not using this argument, just get rid of it (it's available as `table.object` anyway)